api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -67,6 +71,25 @@
   }
 
   @Override
+  public ResponseEntity<Void> createAccount(CreateAccountRequest request) {
+    directoryService.createUser(
+        request.getGivenName(), request.getFamilyName(), request.getUsername(),
+        request.getPassword()
+    );
+    return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
@danqrodney This seems to work. Does it look okay to you?

Reply:
If it works on the client, fine by me.

FWIW: it might be a good idea to return a User object or something from this endpoint eventually, like we do with /me, rather than just a CREATED response. But we could worry about that later.